### ISSUE TITLE
Issue with supported game : Palworld : Filestructure

### ISSUE THREAD

**Issue by Ainweian** [Z1ni/XGP-save-extractor#143]
When executing the extraction, files are generated but not in the expected file structure used by Steam.

I'm trying to export a single player, non-multiplayer flagged world containing 1 character (mine, the host).

A new "Level" folder is generated containing the file 01.sav
The "Level.sav" file is not generated in the base folder

When importing the generated files to the preprepared Steam save folder, the save appears to show correct information regarding the character. But when attempting to load the save, the game forces me to create a new character and fails to load the existing character or base structures.

Generated files and structure : 
![image](https://github.com/Z1ni/XGP-save-extractor/assets/165505205/495098d6-a7ca-48ad-b3ff-8cd9ec1176a9)

**Comment by IRetr0I** [comment]
Im having the same issue, i have also tried renaming but nothing seems to work

**Comment by Klemmnm** [comment]
Has anyone found any solutions? I'm having the same problem.

**Comment by Gaffer13** [comment]
It's the game that does it, not the extractor - you can see the files for yourself. They changed the Game Pass and Xbox save format because they were having issues: 

https://github.com/Z1ni/XGP-save-extractor/issues/136#issue-2177138621

**Comment by k-e-c** [comment]
Yeah, I tried using the extractor and a tool to fix a character and ended up breaking all characters. I'm waiting, hoping there will be a patch that fixes it:
https://www.nme.com/news/gaming-news/latest-palworld-update-is-causing-xbox-save-data-to-be-deleted-3615814
